### PR TITLE
Make(issue_template): Instantiate new Issue template

### PR DESCRIPTION
**Pull request by titom73**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Documentation content changes
- [x] Other (please describe): Github Issue Template engine

## Related Issue(s)

N/A

## Component(s) name

Github Issue Template

## Proposed changes

Switch issue tempalte to new GH engine based on [YAML schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)

## How to test

Please look at this repository to open fake issues: https://github.com/titom73/ansible-avd/issues/new/choose

## Checklist:

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly
- [ ] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      placeholder: Tell us what you see!
+      value: "please detail your issue"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: component
+    attributes:
+      label: Which component of AVD is impacted
+      multiple: false
```
Comment (ClausHolbechArista) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: issues with plugins could impact multiple components.

Comment (titom73) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: We can activate `multiple: true` or also list plugins.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      multiple: true
+      options:
+        - eos_designs
+        - eos_cli_config_gen
+        - cvp_configlet_upload
+        - dhcp_provisioner
+        - eos_config_deploy_cvp
+        - eos_config_deploy_eapi
+        - eos_snapshot
+        - eos_validate_state
```
Comment (carlbuchmann) [inline .github/ISSUE_TEMPLATE/feature-request.yml:32]: Should list all components, and match bug reports?
Also, there should be an option for new_components.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    attributes:
+      label: Use case example
+      value: "please detail your use case"
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-solution
+    attributes:
+      label: Describe the solution you'd like
```
Comment (carlbuchmann) [inline .github/ISSUE_TEMPLATE/feature-request.yml]: update to: "Describe the solution you would like"

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      description: A clear and concise description of what you want to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    id: alternative-solution
+    attributes:
+      label: Describe alternatives you've considered
```
Comment (carlbuchmann) [inline .github/ISSUE_TEMPLATE/feature-request.yml]: update to: "Describe alternatives you have considered"

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      description: A clear and concise description of any alternative solutions or features you've considered.
```
Comment (carlbuchmann) [inline .github/ISSUE_TEMPLATE/feature-request.yml]: update to: "A clear and concise description of any alternative solutions or features you have considered."

**Review by carlbuchmann — APPROVED** [review]
Approved! Some minor feedback to be addressed/discuss, overall this is great!

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 2 above)
Comment (titom73) [reply .github/ISSUE_TEMPLATE/feature-request.yml:32]: Fixed: added `others` and `new components`

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 3 above)
Comment (titom73) [reply .github/ISSUE_TEMPLATE/feature-request.yml]: Fixed

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 4 above)
Comment (titom73) [reply .github/ISSUE_TEMPLATE/feature-request.yml]: Fixed

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 5 above)
Comment (titom73) [reply .github/ISSUE_TEMPLATE/feature-request.yml]: Fixed

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**